### DISCUSSION TITLE
Module ZIP upload - don't extract folder "__MACOSX", skip/ignore this folder

### DISCUSSION THREAD

**Discussion by ShaiMagal** [PrestaShop/PrestaShop#37067] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

If you have module zipped on Mac, you have in zip file folder "__MACOSX" which is totally useless for PrestaShop.

### Expected behavior

We should skip/ignore folder "__MACOSX" and don't extract it.

### Steps to reproduce

1) prepare module with folder __MACOSX (you should use this module for this purpose - [shaim_test.zip](https://github.com/user-attachments/files/17139871/shaim_test.zip))
2) upload it
3) you will find folder "__MACOSX" in "modules" folder.
Totally useless.

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.3

### Your company or customer's name goes here (if applicable).

https://www.openservis.cz/

![Untitled](https://github.com/user-attachments/assets/0a4bcb9b-fa16-4698-91f9-63fb12608236)

**Comment by Hlavtox** [comment]
@ShaiMagal + `.DS_Store`

**Comment by ShaiMagal** [comment]
@Hlavtox Yes, true.

But, maybe we will need to ask ourself, if we don't block everything other than zip name...: HMMM? @matks @kpodemski 
Example file name:
nice_module.zip
So we extract ONLY folder "nice_module" and skip everything else...

**Comment by kpodemski** [comment]
It sounds like a nice improvement, but it's more like an idea than an issue.

**Comment by SharakPL** [comment]
blockwishlist 3.0.2 creates this dir
#37936
